src/nemos/glm.py

```diff
@@ -666,6 +673,12 @@ def fit(
         else:
             data = X
 
+        # check if mask has been set is using group lasso
+        # if mask has not been set, use a single group as default
+        if isinstance(self.regularizer, GroupLasso):
+            if self.regularizer.mask is None:
+                self.regularizer.mask = jnp.ones((1, data.shape[1]))
```

Comment:
is this the same as just doing regular Lasso? then shouldn't we raise an exception because they've probably misunderstood something?

Reply:
Maybe we raise a warning similar to `regularizer_strength`? Point them to the docs and tell them we are reverting back to `Lasso`

Reply:
It is actually the opposite of `Lasso`, it treats all the coeffs as a single group. I would keep it as is, so that the two regularizer defaults have different effects, but in the docstrings should make that clear

Reply:
PS Lasso would be mask=np.eye(data.shape[1])